### PR TITLE
Closes #108 & #109

### LINKED ISSUE #108: Add Minecraft workshop to current events
__Date__ 2017-02-01  
__Time__ 18:00-20:00  
__Location__ TRU, IB1010  
__Link__ http://minecraftmodding.eventbrite.ca

### PR DESCRIPTION
**Pull request by marcolussetti**

### CODE REVIEW

File: events.html [hunk 1]
```diff
@@ -52,7 +52,16 @@
         <div class="col-sm-7 float-right-md-lg">
             <h1>Upcoming Events</h1>
             <div class="event-list">
-                <em>None currently</em>
+                <a class="event" href="https://minecraftmodding.eventbrite.ca">
+                    <h3>Advanced Topics of Java: Minecraft Modding Workshop @ <b>IB1010</b></h3>
+                    <h5>6:00pm on Wednesday, February 1st, 2017</h5>
+                    Would you like a stronger understanding of Java? Do you enjoy playing games? Well now you can have both! Through this seminar you will learn how to use your Java skill to make a mod for Minecraft.
+                </a>
+                <a class="event" href="http://getrust.eventbrite.ca/">
```
Comment (mitchhentges) [inline events.html]: The url should be: `https://getrusty.eventbrite.ca/`

File: events.html [hunk 2]
```diff
@@ -52,7 +52,16 @@
         <div class="col-sm-7 float-right-md-lg">
             <h1>Upcoming Events</h1>
             <div class="event-list">
-                <em>None currently</em>
+                <a class="event" href="https://minecraftmodding.eventbrite.ca">
+                    <h3>Advanced Topics of Java: Minecraft Modding Workshop @ <b>IB1010</b></h3>
```
Comment (mitchhentges) [inline events.html]: Let's put a space between "IB" and "1010", like other events

File: events.html [hunk 3]
```diff
@@ -52,7 +52,16 @@
...
+                    <h5>6:00pm on Wednesday, February 1st, 2017</h5>
+                    Would you like a stronger understanding of Java? Do you enjoy playing games? Well now you can have both! Through this seminar you will learn how to use your Java skill to make a mod for Minecraft.
+                </a>
+                <a class="event" href="http://getrust.eventbrite.ca/">
+                    <h3>Get Rusty Workshop @ <b>IB1010</b></h3>
```
Comment (mitchhentges) [inline events.html]: Same thing: `IB 1010`

**Comment by marcolussetti** [comment]
The changes make sense, but you'll have to approve them (as I am the pull requester).

**Comment by mitchhentges** [comment]
For sure, I am notified when a commit is added to a PR that I am watching

File: events.html [hunk 4]
```diff
@@ -52,7 +52,16 @@
         <div class="col-sm-7 float-right-md-lg">
             <h1>Upcoming Events</h1>
             <div class="event-list">
-                <em>None currently</em>
+                <a class="event" href="https://minecraftmodding.eventbrite.ca">
+                    <h3>Advanced Topics of Java: Minecraft Modding Workshop @ <b>IB 1010</b></h3>
+                    <h5>6:00pm on Wednesday, February 1st, 2017</h5>
+                    Would you like a stronger understanding of Java? Do you enjoy playing games? Well now you can have both! Through this seminar you will learn how to use your Java skill to make a mod for Minecraft.
+                </a>
+                <a class="event" href="http://getrusty.eventbrite.ca/">
```
Comment (mitchhentges) [inline events.html]: For consistency with the above event, can we make this `https://`?

**Pull request merged**